vagkaratzas/proteinfamilies
# This changeset includes a WIP update workflow.

This has to be tested and validated.

High level things to test:
- Overall logic of the update
- Publish files from the updates
- Check that the creation workflow wasn't broken by this

I was using a local set of HMMs and MSAs for testing, pretty much fake data. But the pipeline worked, both ways (create and update).

I left some TODOs and FIXMEs in the code on thigs we should review.

There are many changes on python scripts because I ran ruff and black, which are linting and formatting tools.
